campusromero_openedx_extensions/custom_registration_form/forms.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, *args, **kwargs):
             "invalid": _('This day number is not valid, please check your input.'),
         }
         self.fields['dni'].error_messages = {
+            'required': _('Please enter DNI number.'),
```

Comment:
Please check this translation is still present

Reply:
yes, they are still present

Reply:
it's still present https://github.com/eduNEXT/campusromero-openedx-extensions/blob/master/campusromero_openedx_extensions/locale/es_419/LC_MESSAGES/django.po#L33